### ISSUE TITLE
BUG: Screencast mode onlyKeyboardShortcuts is not working

### ISSUE THREAD

**Issue by jaktestowac** [microsoft/vscode#179096]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:  1.77.0
- OS Version:  Windows 11 Pro 22H2
This bug is regression introduced in current version.

Steps to Reproduce:

1. Set screencastMode.onlyKeyboardShortcuts  to true
2. Toggle Screencast Mode on
3. Observed: Single keyboard entries are presented (letters numbers). Changing screencastMode.onlyKeyboardShortcuts not works.
5. Expected: single keyboard entries to be hidden (https://code.visualstudio.com/updates/v1_38?ref=codebldr#_screencast-mode)

**Comment by joaomoreno** [comment]
The description of that setting is:

> Show only keyboard shortcuts in screencast mode (**do not include action names**).

If single keyboard entries are presented, it's doing what it's supposed to. This setting just controls whether to show action/command names or not.

**Comment by jaktestowac** [comment]
> The description of that setting is:
> 
> > Show only keyboard shortcuts in screencast mode (**do not include action names**).
> 
> If single keyboard entries are presented, it's doing what it's supposed to. This setting just controls whether to show action/command names or not.

So there is some misunderstanding in screencast features (is there any documentation for it?). Referring to this comment: https://github.com/microsoft/vscode/issues/108951#issuecomment-717870955
This setting was made to display **Only Keyboard Shortcut** and do not display single items. 
And it worked this way in previous release. So why change was introduced?  I believe that the mixup with this drop-down causing confusion:
  
![obraz](https://user-images.githubusercontent.com/72373858/231166388-ea56f005-6fc2-4602-b209-533a02890c52.png)

This window was not working as designed in 1.76 and was fixed. Now shortcuts and commands are displayed separately or together (just checked 1.76 and this was broken).
Tried with Ctrl+Shift+P - works fine. So **Only Keyboard Shortcut** will be redundant if we want to use it just without action (commands) names (this feature provide mentioned dropdown window)

To sum up. In 1.77 the **Keyboard Shortcuts Format** was fixed but **Only Keyboard Shortcut** was broken. 

One thing what need to clarification if **action** term in **Only Keyboard Shortcut** in is the same as **commands** in **Keyboard Shortcuts Format**. If there a difference please elaborate.
